bindings/nodejs/generated/main.cc.prologue

```diff
@@ -8,5 +9,6 @@ using namespace v8;
 #include "../src/common.h"
 
 NAN_MODULE_INIT(Init) {
+  sol_init();
```

Comment:
before doing the init, you should specify your libuv mainloop implementation, ok? (when you integrate mainloops).

https://github.com/solettaproject/soletta/blob/master/src/lib/common/include/sol-mainloop.h#L1013

You will need to define a `struct sol_mainloop_implementation libuv_soletta_mainloop_impl` that will define wrappers to translate our primitives to libuv. If the signatures of our callbacks do not match, then you need to translate back when calling them. In a way it is similar to our Glib main loop implementation (similar in concept, not equal in implementation):  https://github.com/solettaproject/soletta/blob/master/src/lib/common/sol-mainloop-impl-glib.c